README.md

```diff
@@ -102,28 +105,113 @@ Options:
 ./target/release/databroker-perf [OPTIONS]
 ```
 
+## Default test result output
+
+By default, the group results output will be summarised and contracted as follows:
+```
+Global Summary:
+  API: KuksaValV2
+  Total elapsed seconds: 6
+  Skipped test seconds: 2
+  Total signals: 8 signals
+  Sent: 2283 signal updates
+  Skipped: 768 signal updates
+  Received: 1515 signal updates
+  Signal/Second: 378 signal/s
+  95% in under: 0.907 ms
+  Fastest:   0.252 ms
+  Slowest:   1.349 ms
+  Average:   0.732 ms
+
+Latency histogram:
+    0.248 ms [7    ] |
+    0.347 ms [20   ] |
+    0.446 ms [7    ] |
+    0.545 ms [196  ] |∎
+    0.644 ms [408  ] |∎∎∎∎
+    0.743 ms [378  ] |∎∎∎
+    0.842 ms [292  ] |∎∎
+    0.941 ms [146  ] |∎
+    1.040 ms [47   ] |
+    1.139 ms [9    ] |
+    1.238 ms [4    ] |
+    1.337 ms [1    ] |
+
+Latency distribution:
+  10% in under 0.576 ms
+  25% in under 0.621 ms
+  50% in under 0.717 ms
+  75% in under 0.829 ms
+  90% in under 0.915 ms
+  95% in under 0.970 ms
+  99% in under 1.069 ms
+```
+
+For a detailed output of the results, please enable the corresponding flag like:
+
+```
+./target/release/databroker-perf --detailed-output
+```
+
+## Group config file
```

Comment:
This is now called "test data file", not "config file", right? ( As supported argument changed)